src/getCoinCombination.test.js

```diff
@@ -7,4 +7,28 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  test('Should return the smallest coin combination for 1 cent', () => {
```

Comment:
accotding to the test names, all test are the same. Try to rename your test in the suggested way
```suggestion
  test('Should return array with 1 peny when cents is equal to 1', () => {
```

Reply:
Thanks for the help)
fixed